services/QuillLMS/client/app/bundles/Teacher/components/classrooms/import_clever_classroom_students_modal.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react'
+import Pusher from 'pusher-js';
+
+import ButtonLoadingIndicator from '../shared/button_loading_indicator'
+import { requestPut } from '../../../../modules/request/index.js';
+
+const smallWhiteCheckSrc = `${process.env.CDN_URL}/images/shared/check-small-white.svg`
+
+interface ImportCleverClassroomStudentsModalProps {
+  close: () => void;
+  onSuccess: (snackbarCopy: string) => void;
+  classroom: any;
+}
+
+interface ImportCleverClassroomStudentsModalState {
+  checkboxOne?: boolean;
+  waiting: boolean;
+}
+
+export default class ImportCleverClassroomStudentsModal
+  extends React.Component<ImportCleverClassroomStudentsModalProps, ImportCleverClassroomStudentsModalState> {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      checkboxOne: false,
+      waiting: false
+    }
+
+    this.handleToggleCheckbox = this.handleToggleCheckbox.bind(this)
+    this.handleImportStudents = this.handleImportStudents.bind(this)
```

Comment:
I know you did a ton of work refactoring files like `active_classrooms`, and that this is based on an existing file, but since it's new code it does seem to me like we should probably avoid breaking linter rules as much as possible (like these binds here, and calling `this.props.onSuccess` below, etc).